DESCRIPTION

```diff
@@ -51,6 +52,7 @@ Suggests:
     digest,
     fansi,
     fs,
+    httptest,
```

Comment:
I used this and environmental variables for tests because I saw too many problems with mockery's depth argument.